charts/librephotos/values.yaml

```diff
@@ -147,7 +147,8 @@ secret:
   ADMIN_USERNAME: "admin"
   ADMIN_PASSWORD: "password"
   MAPBOX_API_KEY: ""
-
+  existingSecret: false
```

Comment:
This is going to add an extra line in the Secret . as the secret iterates over the hash
It needs a bit different approach, I would say.

I want to also add support, for autogenerated Admin password (first install).
Do you want to work on improving this, or want me to do so and post a pr?

Reply:
you also miss the deployment changes, as deployment needs to use the secret as optional or not use declare it at all.

Reply:
so i sent a small set of follow up changes i ve moved the existing secret one level up in the values:
```
secret:
  ADMIN_EMAIL: "admin@mydomain.com"
  ADMIN_USERNAME: "admin"
  ADMIN_PASSWORD: "password"
  MAPBOX_API_KEY: ""

# instead use an existingSecret: "librephotos-creds"
existingSecret: false
```

that way it wouldnt end up in the secrets in the iteration, and the structure doesnt change for users already using the chart. and i ve added a small if to also flip the secret in the deployment.

I would most likely use an externalSecret so t ome the autogeneration is not needed, however it's definately a good feature you should add ( any password is better then password :D )